### PR TITLE
Base part of hometask for Lec 11 with failed test for links

### PR DESCRIPTION
**Pull request by AlesiaRebkavets**

### CODE REVIEW

File: Lecture11/Tests.cs [hunk 1]
```diff
@@ -73,6 +75,175 @@ public void OneTimeSetup()
...
+            homeCollapseButton.Click();   // clicking the button
+            var documentsCollapseButton = _driver.FindElement(By.XPath("//label[@for=\"tree-node-documents\"]/../button")); // finding documentsCollapseButton
+            documentsCollapseButton.Click(); // clicking the button
+            var officeCollapseButton = _driver.FindElement(By.XPath("//label[@for=\"tree-node-office\"]/../button")); // finding officeCollapseButton
+            officeCollapseButton.Click();   // clicking the button
+            var privateCheckbox = _driver.FindElement(By.XPath("//label[@for=\"tree-node-private\"]/span[1]")); // finding privateCheckbox
+            privateCheckbox.Click();   // selecting the element;
+            var classifiedCheckbox = _driver.FindElement(By.XPath("//label[@for=\"tree-node-classified\"]/span"));  // finding classifiedCheckbox
+            classifiedCheckbox.Click();  // selecting the element;
+            var youHaveSelectedLabelText = _driver.FindElement(By.XPath("//span[text()=\"private\"]")).Text + " " + _driver.FindElement(By.XPath("//span[text()=\"classified\"]")).Text;  // saving the displayed final text to a separate variable
```
Comment (ezhukovskaya) [inline Lecture11/Tests.cs]: replace concatenation with interpolation

File: Lecture11/Tests.cs [hunk 2]
```diff
@@ -73,6 +75,175 @@ public void OneTimeSetup()
...
+        [Test]
+        public void ButtonsClickTest()     //test for Buttons
+        {
+            _driver.Navigate().GoToUrl("https://demoqa.com/buttons");         // opens the indicated url
+
+            var doubleClickButton = _driver.FindElement(By.Id("doubleClickBtn"));       // saving locators for the buttons
+            var rightClickButton = _driver.FindElement(By.Id("rightClickBtn"));
+            var clickButton = _driver.FindElement(By.XPath("//button[text()=\"Click Me\"]"));
+
+            _driverActions.DoubleClick(doubleClickButton).Perform();      // performing double click
```
Comment (ezhukovskaya) [inline Lecture11/Tests.cs]: better to use Build().Perform()
fix everywhere

File: Lecture11/Tests.cs [hunk 3]
```diff
@@ -1,6 +1,8 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
```
Comment (ezhukovskaya) [inline Lecture11/Tests.cs]: clean usings

**Review by ezhukovskaya — CHANGES_REQUESTED** [review]

File: Lecture11/Tests.cs [hunk 4]
```diff
@@ -73,6 +75,175 @@ public void OneTimeSetup()
...
+            var AddButton = _driver.FindElement(By.Id("addNewRecordButton"));   // finding "Add" button element
+            AddButton.Click();    // clicking the "Add" button
+            var registrationForm = _driver.FindElement(By.Id("registration-form-modal"));  // finding registration form modal label
+
+            Assert.IsTrue(registrationForm.Displayed);   // verifying if "Registration Form" modal opened
+
+            var FirstNameTextBox = _driver.FindElement(By.Id("firstName"));   // finding elements of registration form
+            var LastNameTextBox = _driver.FindElement(By.Id("lastName"));
+            var EmailTextBox = _driver.FindElement(By.Id("userEmail"));
+            var AgeTextBox = _driver.FindElement(By.Id("age"));
```
Comment (ezhukovskaya) [inline Lecture11/Tests.cs]: change name to ageTextBox

**Review by ezhukovskaya — CHANGES_REQUESTED** [review]

File: Lecture11/Tests.cs (hunk 4 above)
Comment (AlesiaRebkavets) [reply Lecture11/Tests.cs]: с маленькой буквы, исправить

File: Lecture11/Tests.cs (hunk 1 above)
Comment (AlesiaRebkavets) [reply Lecture11/Tests.cs]: done

File: Lecture11/Tests.cs (hunk 2 above)
Comment (AlesiaRebkavets) [reply Lecture11/Tests.cs]: done

File: Lecture11/Tests.cs (hunk 3 above)
Comment (AlesiaRebkavets) [reply Lecture11/Tests.cs]: done

File: Lecture11/Tests.cs (hunk 4 above)
Comment (AlesiaRebkavets) [reply Lecture11/Tests.cs]: done

**Review by ezhukovskaya — APPROVED** [review]

**Pull request merged**